### PR TITLE
all: use strings.Contains instead Index

### PR DESCRIPTION
**Pull request by lelenanam**
**\- What I did**
Use  strings.Contains instead strings.Index
**\- How I did it**

**\- How to verify it**
Compile docker daemon.
**\- Description for the changelog**

**\- A picture of a cute animal (not mandatory but encouraged)**
cat

### CODE REVIEW

**Comment by LK4D4** [comment]
LGTM

**Comment by runcom** [comment]
LGTM

File: pkg/fileutils/fileutils.go [hunk 1]
```diff
@@ -180,7 +180,7 @@ func regexpMatch(pattern, path string) (bool, error) {
 		} else if ch == '?' {
 			// "?" is any char except "/"
 			regStr += "[^" + escSL + "]"
-		} else if strings.Index(".$", string(ch)) != -1 {
+		} else if strings.Contains(".$", string(ch)) {
```
Comment (aaronlehmann) [inline pkg/fileutils/fileutils.go]: Perhaps `ch == '.' || ch == '$'` would be a bit clearer in this case?

Comment (morelena) [inline pkg/fileutils/fileutils.go]: PTAL

**Comment by aaronlehmann** [comment]
LGTM

**Review by thaJeztah — APPROVED** [review]
LGTM

all green, merging

thanks @morelena!

**Pull request merged**